lib/endpoints/class-wp-rest-comments-controller.php

```diff
@@ -700,9 +700,9 @@ public function get_item_schema() {
 					'type'         => 'integer',
 					'context'      => array( 'view', 'edit', 'embed' ),
 					),
-				'author_avatar_url' => array(
-					'description'   => 'Avatar URL for the object author.',
-					'type'          => 'string',
+				'author_avatar_urls' => array(
+					'description'   => 'Avatar URLs for the object author.',
+					'type'          => 'array',
 					'format'        => 'uri',
```

Comment:
You're actually returning an object now, so it would be an object with specific properties